cmake/CPackOptions.cmake.in

```diff
@@ -42,13 +42,10 @@ elseif(CPACK_GENERATOR STREQUAL "RPM")
   set(CPACK_RPM_PRE_INSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/cmake/CPack_preinstall.sh.in")
   set(CPACK_RPM_POST_INSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/cmake/CPack_postinstall.sh.in")
   set(CPACK_RPM_PRE_UNINSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/cmake/CPack_preuninstall.sh.in")
+  set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "/etc/profile.d/cubrid.csh /etc/profile.d/cubrid.sh /etc/init.d/cubrid /etc/init.d/cubrid-ha")
   set(CPACK_RPM_USER_FILELIST
     "%config(noreplace) /opt/cubrid/conf"
-    %dir "/opt/cubrid/databases"
-    "/etc/profile.d/cubrid.csh"
-    "/etc/profile.d/cubrid.sh"
-    "/etc/init.d/cubrid"
-    "/etc/init.d/cubrid-ha"
+    "%dir /opt/cubrid/databases"
```

Comment:
Adding the %attr directive will eliminate the compile error in CPACK_RPM_USER_FILELIST in CPackOptions.cmake.
Tested in cmake 2.8.12.2, 3.26.0

```
set(CPACK_RPM_USER_FILELIST
    "%config(noreplace) /opt/cubrid/conf"
    "%dir /opt/cubrid/databases"
    "%attr(750,root,root) /etc/init.d/cubrid"
    "%attr(750,root,root) /etc/init.d/cubrid-ha"
    "%attr(644,cubrid,root) /etc/profile.d/cubrid.csh"
    "%attr(644,cubrid,root) /etc/profile.d/cubrid.sh"
    )
```